tests/forms_tests/tests/test_validators.py

```diff
@@ -8,38 +8,36 @@
 from django.core.exceptions import ValidationError
 
 
-class UserForm(forms.Form):
```

Comment:
I'd like if you made this change in a separate commit so it doesn't confuse this one.